app/javascript/components/recipes/new/add-button.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+  <button
+    @click="clickButton"
+  >
+    + 
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Trailing spaces not allowed  no-trailing-spaces